packages/utilities/psammead-locales/moment/ps.js

```diff
@@ -0,0 +1,129 @@
+// moment.js locale configuration
+// locale: Pashto (ps)
+
+const moment = require('moment');
+const jalaaliHelper = require('./helpers/jalaali');
+
+const symbolMap = {
+  '1': '۱',
+  '2': '۲',
+  '3': '۳',
+  '4': '٤',
+  '5': '۵',
+  '6': '۶',
+  '7': '۷',
+  '8': '۸',
+  '9': '۹',
+  '0': '۰',
+};
+
+const numberMap = {
+  '۱': '1',
+  '۲': '2',
+  '۳': '3',
+  '٤': '4',
+  '۵': '5',
+  '۶': '6',
+  '۷': '7',
+  '۸': '8',
+  '۹': '9',
+  '۰': '0',
+};
+
+// pashto calendar months
+const pashtoMonths = [
+  'وری',
+  'غویی',
+  'غبرګولی',
+  'چنګاښ',
+  'زمری',
+  'وږی',
+  'تله',
+  'لړم',
+  'لیندۍ',
+  'مرغومی',
+  'سلواغه',
+  'کب',
+];
+
+const jalaliFromats = [
+  'HH:mm', // LT
+  'HH:mm:ss', // LTS
+  'D MMMM YYYY', // LL FROMAT
+  'D MMMM YYYY HH:mm', // LLL FROMAT
+  'dddd, D MMMM YYYY HH:mm', // LLLL FROMAT
+];
+moment.defineLocale('ps', {
+  months: 'جنوري_فبروري_مارچ_اپریل_می_جون_جولاې_اګست_سپتمبر_اکتوبر_نومبر_ډیسمبر'.split(
+    '_',
+  ),
+  monthsShort: 'جنوري_فبروري_مارچ_اپریل_می_جون_جولاې_اګست_سپتمبر_اکتوبر_نومبر_ډیسمبر'.split(
```

Comment:
What’s your source for these translations?

Reply:
Hey Drew :) , these translations are the same in the `moment-ws` repo here [https://github.com/bbc/moment-ws/blob/master/src/locale/ps.js](url)